engine-comparison/runner.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Copyright 2017 Google Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+
+# set -x
+. benchmark.cfg
+. parameters.cfg
+. fengine.cfg
+
+FENGINE_NAME=$(curl http://metadata.google.internal/computeMetadata/v1/instance/attributes/fengine -H "Metadata-Flavor: Google")
+
+BINARY=${BENCHMARK}-${FUZZING_ENGINE}
+
+rm -fr corpus
+mkdir corpus
+mkdir results
```

Comment:
This was a shortcut, only because all of results is currently rewritten with `>` not `>>`. But in principle it should be reset as well